src/app/shared/components/config-cards/twitchbot-config/twitchbot-config.component.html

```diff
@@ -0,0 +1,73 @@
+<div class="title-row">
+  <h2 class="mat-h2 title-name">Twitch Bot Config</h2>
+
+  <mat-icon svgIcon="twitch"></mat-icon>
+</div>
+
+<div>
+  <form [formGroup]="form">
+    <mat-tab-group>
+      <mat-tab label="Authentication">
+
+        <p>To obtain the bot token, login into twitch with your bot account, click on the "Get Twitch Token" button and
+          follow the steps in the window that opens.<br /></p>
+        <p><br /><a mat-button mat-raised-button href="https://twitchapps.com/tmi/" target="_blank"
+                    class="twitchSetting__token-button">
+          <mat-icon svgIcon="launch"></mat-icon> &nbsp;Get Twitch Token</a></p>
+
+        <mat-form-field *ngIf="config$ | async as config">
```

Comment:
move `*ngIf="config$ | async as config"`that to the form element that way you dont need to use it on every form-field